### PR TITLE
setup database

### PR DESCRIPTION
**Pull request by Khoacannotcode**
## Related Tickets:
- [#45187](https://edu-redmine.sun-asterisk.vn/issues/45187)

## WHAT this PR does?
- Create migration for setting up all table. Including: users, roles, posts, images, comments, categories.

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by Khoacannotcode** [comment]
ready

File: database/migrations/2022_01_18_024625_create_posts_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments("id");
```
Comment (ghost) [inline database/migrations/2022_01_18_024625_create_posts_table.php]:
```suggestion
            $table->increments('id');
```
mỗi dòng nháy 1 kiểu thế

Comment (Khoacannotcode) [reply database/migrations/2022_01_18_024625_create_posts_table.php]: Vâng ạ, em sửa lại ngay

**Comment by Khoacannotcode** [comment]
ready

**Pull request merged**